schema.sql

```diff
@@ -0,0 +1,23 @@
+DROP DATABASE IF EXISTS sweethornbillbookstore;
+CREATE DATABASE sweethornbillbookstore;
+
+\c sweethornbillbookstore;
+
+
+CREATE TABLE books (
+  id SERIAL PRIMARY KEY,
+  title VARCHAR,
+  author VARCHAR,
+  genre VARCHAR,
+  description VARCHAR,
+  img_url VARCHAR
+);
+
+INSERT INTO books ( title, author, genre, description, img_url )
```

Comment:
i would separate these out into a fixture-data.sql file so that this file only handles schema creation.